components/Cases/Cases.jsx

```diff
@@ -3,17 +3,29 @@ import PropTypes from 'prop-types';
 
 import CasesTable from 'components/Cases/CasesTable';
 import ErrorMessage from 'components/ErrorMessage/ErrorMessage';
+import ErrorSummary from 'components/ErrorSummary/ErrorSummary';
 import Button from 'components/Button/Button';
 import Spinner from 'components/Spinner/Spinner';
 import { getCasesByResident } from 'utils/api/cases';
+import { getResident } from 'utils/api/residents';
 
 const Cases = ({ id }) => {
   const [error, setError] = useState();
   const [loading, setLoading] = useState(false);
+  const [personRestricted, setPersonRestricted] = useState();
   const [results, setResults] = useState();
+  const getPerson = async () => {
```

Comment:
We can pass down `person` from `PersonView` as a render prop, as we did in the `record` page.